python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -179,6 +179,7 @@ def create(
         map_task_kwargs: Optional[Dict[str, Any]] = None,
         ray_remote_args_fn: Optional[Callable[[], Dict[str, Any]]] = None,
         ray_remote_args: Optional[Dict[str, Any]] = None,
+        per_block_limit: Optional[int] = None,
```

Comment:
```suggestion
        total_num_rows_limit: Optional[int] = None,
```

Reply:
Sorry, for the back and forth, but on a second thought now per-block limit is a misnomer (since we're not limiting per block, but for the whole output)

Please update everywhere